services/src/main/java/org/keycloak/protocol/oid4vc/issuance/VCIssuerException.java

```diff
@@ -0,0 +1,17 @@
+package org.keycloak.protocol.oid4vc.issuance;
+
+/**
+ * Exception to be thrown in case credentials issuance fails.
+ *
+ * @author <a href="https://github.com/wistefan">Stefan Wiedemann</a>
+ */
+public class VCIssuerException extends RuntimeException {
+
+	public VCIssuerException(String message) {
```

Comment:
Minor: Is it possible to use same padding like other Keycloak classes? So 4 characters only in java classes?

Reply:
fixed by #f74153f